src/js/stories/table.css

```diff
@@ -0,0 +1,11 @@
+.red {
+    color: #ff0000;
+}
+
+.blue {
+    color: #0000ff;
+}
+
+.green {
+    color: #00ff00;
+}
```

Comment:
I really don't like having storybook-specific CSS, nor bringing in all the magic in the accompanying loaders. The point of storybooks should be to demonstrate how things will actually be, and we've got styles in the JDL already. There may be a good use case for this that I haven't thought of though, so let me know if that's the case.